src/robot_dart/robot_dart_simu.cpp

```diff
@@ -145,6 +145,29 @@ namespace robot_dart {
         _old_index = index;
     }
 
+    void RobotDARTSimu::step_once()
```

Comment:
For removing duplicate code, I will use this function inside the `run()` function.